src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -3,40 +3,113 @@
 import static java.util.Objects.requireNonNull;
 
 import seedu.address.commons.core.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
-import seedu.address.model.person.NameContainsKeywordsPredicate;
+import seedu.address.model.person.predicates.NameContainsKeywordsPredicate;
+import seedu.address.model.person.predicates.StaffHasCorrectIndexPredicate;
 
 /**
  * Finds and lists all persons in address book whose name contains any of the argument keywords.
- * Keyword matching is case insensitive.
+ * Keyword matching is case-insensitive.
  */
 public class FindCommand extends Command {
 
     public static final String COMMAND_WORD = "find";
+    public static final String COMMAND_TAG_INDEX = "-i";
```

Comment:
I believe this belongs in CLISyntax?

Reply:
CLI syntax has the "n/" format though, will it not get confused?